frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -190,7 +190,7 @@ export const NavBar = withRouter<NavBarProps>(({ pages, baseURL, basePath }) =>
           'co-m-horizontal-nav-item--active': matchURL?.isExact,
         });
         return (
-          <li className={klass} key={nameKey || name}>
+          <li className={klass} key={nameKey || href}>
```

Comment:
Should we simply always use `href` and never name?

On line 186, we have a `path || href` check... Is `href` always defined?

Reply:
Href can be an empty string, but that only appears to happen in for the "Details" tab.   Href, while in theory, could be undefined, the tab would be non-functional by pointing to a url like: ".../undefined". 

On the other hand, path appears to be undefined on a regular basis which might explain why the check was put in on line 186.

Reply:
We could put in `<li className={klass} key={nameKey || href || name}>` to cover all the bases, but I'm not sure it is neccessary.

As a side note, it looks like the exact instance that is referenced in this bug no longer exists (because the names are different), but I still think we should fix this.

Reply:
I think the empty string is a valid key? In which case `key={href}` should work and should be unique.

Reply:
@spadgett An empty string is a valid key (I double-checked to be sure because I was doubting my memory).  I agree, let's just make href the key. Yay simplification.  

New changes coming soon.